data/brands/amenity/parcel_locker.json

```
@@ -990,6 +990,17 @@
         "parcel_mail_in": "yes",
         "parcel_pickup": "yes"
       }
+    },
+    {
+      "displayName": "Yeep",
+      "locationSet": {"include": ["gb"]},
+      "matchNames": ["pudo","pick up, drop off"],
+      "tags": {
+        "amenity": "parcel_locker",
+        "name": "Yeep",
+        "brand": "Yeep",
+        "brand:wikidata": "Q123421114"
```

```suggestion
        "brand:wikidata": "Q123421114",
        "parcel_mail_in": "yes",
        "parcel_pickup": "yes"
```
per https://yeeplockers.com/lockers/